website/example/Modal.demo.js

```diff
@@ -26,55 +26,57 @@ const Example = () => {
           />
         </span>
       </div>
-      <Modal
-        fullContent
-        title="John Doe"
-        isOpen={open}
-        isDismissable
-        isClosable
-        size="large"
-        onClose={() => setOpen(!open)}
-      >
-        <>
-          <header className="ex-modal__header">
-            <div className="ex-modal__profile">
-              <div className="ex-modal__avatar" />
-              <h2>John Doe</h2>
-            </div>
-            <IconButton
-              appearance="ghost"
-              size="small"
-              icon={<Close />}
-              onClick={() => setOpen(!open)}
-            />
-          </header>
-          <h4>Your personal info</h4>
-          <form className="ex-modal__form">
-            <div className="ex-modal__formGroup">
-              <HelperText>First name</HelperText>
-              <Input value="John" />
-            </div>
-            <div className="ex-modal__formGroup">
-              <HelperText>Last name</HelperText>
-              <Input value="Doe" />
-            </div>
-            <div className="ex-modal__formGroup">
-              <HelperText>Job title</HelperText>
-              <Input placeholder="e.g. Designer" />
-            </div>
-            <div className="ex-modal__formGroup">
-              <HelperText>Email</HelperText>
-              <Input disabled value="john.doe@acm.com" />
-            </div>
-          </form>
-          <footer className="ex-modal__footer">
-            <Button onClick={() => setOpen(!open)} appearance="secondary">
-              Cancel
-            </Button>
-            <Button disabled>Update profile</Button>
-          </footer>
-        </>
-      </Modal>
+      {open && (
```

Comment:
Does the animation of the model still work if you add this? This if statement is inside the modal component

Reply:
the annimation still works yes.